src/content/docs/tutorial-create-alerts/create-chart-alerts.mdx

```diff
@@ -0,0 +1,121 @@
+
+4. Choose whether or not to use **Evaluation delay**. This option enables a custom period that New Relic waits before starting to evaluate a newly ingested signal against the threshold you set. It's useful for preventing false positive incidents when a data stream begins to report. For your first alert, you can leave it to the disabled toggle unless you know that this option applies to your data.
+
+5. Select **Next**.
+    </Step>
+    <Step>
+## Set the static thresholds
+
+<img
+    title="Set static thresholds"
+    alt="A screenshot displaying how to set static thresholds for an alert condition"
+    src={alertsThresholds}
+/>
+
+1. Start by selecting whether the threshold will have a status of **Critical** or **Warning** in the **Security level** section.
+
+2. Set the limit that, when met, will open an incident. These values will change depending on what you want the condition to do. For example, if you wanted to create a condition for an error message, you might set the threshold to **equal to 1 at least once in 5 minutes**, but if you wanted to create a condition for longer periods of latency issues, you might set the threshold to **above or equal to 50 for at least 15 minutes**.
+
+3. If you want to add another threshold, select either **Add threshold** or **Add lost signal threshold**, depending on your needs. Lost signal thresholds are important for monitoring entities that may stop reporting or queries that may [return null values](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/#query-order).
+
+4. Select **Next**
+    </Step>
+    <Step>
+## Add alert condition details
+
+<img
+    title="Alert condition details"
+    alt="A screenshot displaying how to create the details for an alert"
+    src={alertsDetails}
+/>
+
+1. Add a name for your alert in the **Name** field. This will help you identify this condition, as well as find the condition in the future in case you need to edit or remove it.
+
+2. Choose the policy for this condition to connect to. If you have an **Existing policy**, select it from the drop-down. If not, you'll need to [create a policy](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/#alert-policy-name) by selecting **New policy** and creating it from here.
+
+3. Set any of the additional settings for the alert. This includes setting how long until open incidents are automatically closed and entering any [custom descriptions](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/alert-custom-violation-descriptions/) for the incidents.
+
+4. Select **Save condition** to finish.
+    </Step>
+</Steps>
+
+You've now created an alert condition, and helped to make identifying symptoms within your system that much easier! While there are many options to select when creating an alert condition, the basic process when starting from a chart will always follow the steps above. 
```

Comment:
```While there are many options to select when creating an alert condition```

There sure are! It's a PITA TBH—but I worry this conveys that it's a PITA.

Can we reframe this as a positive? Or, can we reframe as a positive _and_ mention it beforehand? Something like:


```We offer lots of configuration options for alerts, so you can fine-tune them for your team and whatever you're monitoring. Feel free to go with the default values for your first go, or check out the steps below for in-depth explanations and recommendations.```

Reply:
Good call and agreed, I'll make the suggested change. Thanks!